pkg/controller/configmap/config.go

```diff
@@ -71,6 +71,7 @@ func (adaptor *ConfigMapK8s) FillDefaults(configmap *corev1.ConfigMap, spec *con
 		return err
 	}
 	appendErrorIfNotNil(&errs, fillDefault(cfg, "coe", "adaptor", "kubernetes", true))
```

Comment:
Just want to confirm that whether we should remove this line?  @timdengyun @dantingl 
I didn't remove this in my patch as I thought `coe.adaptor` should be filled and then rendered to ncp configmap.